app/controllers/api/AtomRepresentation.java

```diff
@@ -0,0 +1,160 @@
+            "xmlns:" + DATATYPE_REGISTER_NAMESPACE_PREFIX + "=\"http://fields.openregister.org/datatype/\"";
+
+    public static final String TEXT_ATOM = "application/atom+xml; charset=utf-8";
+    public static final String RFC3339_DATETIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZZ";
+    private final CurieResolver curieResolver;
+
+    public AtomRepresentation() {
+        curieResolver = new CurieResolver(ApplicationConf.getRegisterServiceTemplateUrl());
+    }
+
+    @Override
+    public Result toListOfRecords(Register register, List<Record> records, Map<String, String[]> requestParams, Map<String, String> representationsMap, String previousPageLink, String nextPageLink) {
+        String atomHeader = createAtomHeader(register, records);
+        String entries = records.stream()
+                .map(r -> renderRecord(r, register))
+                .collect(Collectors.joining());
+        String atomFooter = createAtomFooter();
+        return ok(atomHeader + entries + atomFooter).as(TEXT_ATOM);
+    }
+
+    @Override
+    public Result toRecord(Register register, Record record, Map<String, String[]> requestParams, Map<String, String> representationsMap, List<RecordVersionInfo> history) {
+        String atomHeader = createAtomHeader(register, Collections.singletonList(record));
+        String entry = renderRecord(record, register);
+        String atomFooter = createAtomFooter();
+
+        return ok(atomHeader + entry + atomFooter).as(TEXT_ATOM);
+    }
+
+    @Override
+    public boolean isPaginated() {
+        return false;
+    }
+
+    private String createAtomHeader(Register register, List<Record> records) {
+        DateTime mostRecentlyUpdated = mostRecentlyUpdated(records);
+
+        return "<feed " + FIELD_REGISTER_NAMESPACE + "\n" +
+                " " + DATATYPE_REGISTER_NAMESPACE + "\n" +
+        " xmlns=\"http://www.w3.org/2005/Atom\">\n" +
+                " <title>" + "TODO" + "</title>\n" +
+                " <id>" + curieResolver.resolve(new Curie(register.name(), "latest.atom")) + "</id>\n" +
+                "<link rel=\"self\" href=\"" + curieResolver.resolve(new Curie(register.name(), "")) + "\" />\n" +
+                "<updated>" + mostRecentlyUpdated.toString(RFC3339_DATETIME_FORMAT) + "</updated>\n" +
+                "<author><name>openregister.org</name></author>\n";
+    }
+
+    private DateTime mostRecentlyUpdated(List<Record> records) {
+        Optional<Record> mostRecentlyUpdatedRecordO = records.stream()
+                .max((a, b) ->
+                        Long.valueOf(a.getMetadata().get().creationTime.getMillis() - b.getMetadata().get().creationTime.getMillis()).intValue());
+        if(mostRecentlyUpdatedRecordO.isPresent()){
+            return mostRecentlyUpdatedRecordO.get().getMetadata().get().creationTime;
+        }
+
+        // Every record should have a metadata with a creationtime - if not something bad has happened and we shouldnt carry on.
+        return null;
```

Comment:
if record is not present than there will a NullPointerException as result of returning null. If that should never happen than we can calll ".get()" directly on Optional.